src/application/worker/loop.js

```diff
@@ -149,7 +149,11 @@ function loop(delta, features) {
 
       const module = active[group.modules[j]];
 
-      if (!module.meta.enabled || module.meta.alpha < 0.001) {
+      if (
+        module === undefined ||
```

Comment:
This is probably an okay check, but we really shouldn't be encountering an undefined module here.
What's the case that prompted the check to be added here? We should fix the cause.

Reply:
I tested this again and the problem comes up when two modules share the same name.

## Steps to reproduce
* Create a module called `Lines` in the in `modV/media/default/module` of type `shader`
* Load the branch `feat/541/vidvoxIsfModules` which contains an ISF module called `Lines`
* See these errors:

```
modules.js#removeActiveModule: Cannot destructure property 'meta' of 'module' as it is undefined.

loop.js: Cannot read property 'meta' of undefined

loop.js: Cannot destructure property 'meta' of 'module' as it is undefined.

modules.js#init: Cannot read property '$moduleName' of undefined
```

Not all of them are shown all the time, but if the module-name-conflict is resolved, none of them are shown. 

What I also see is this warning a couple of times:

```
Did not remove inputLink. Could not find input with id
```

Reply:
@TimPietrusky Yes I agree.

This sounds like a bug in the module creation. I'm surprised naming conflicts still happen, but let's create a new ticket as you suggest and remove the check for undefined here.

Reply:
See #600